sdks/java/io/parquet/src/main/java/org/apache/beam/sdk/io/parquet/ParquetIO.java

```diff
@@ -906,7 +903,7 @@ public BlockTracker(OffsetRange range, long totalByteSize, long recordCount) {
         }
       }
 
-      public void makeProgress() throws Exception {
+      public void makeProgress() throws IOException {
```

Comment:
Using a more specific exception in lieu of broad `Exception`

Reply:
good